src/components/notifications/SMSIR.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+    <div class="mb-3">
+        <label for="smsir-key" class="form-label">{{ $t("API Key") }}</label>
+        <HiddenInput id="smsir-key" v-model="$parent.notification.smsirApiKey" :required="true" autocomplete="new-password"></HiddenInput>
+    </div>
+    <div class="mb-3">
+        <label for="smsir-number" class="form-label">{{ $t("Recipient Number") }}</label>
+        <input id="smsir-number" v-model="$parent.notification.smsirNumber" placeholder="9123456789" type="text" maxlength="10" minlength="10" class="form-control" required>
+    </div>
+    <div class="mb-3">
+        <label for="smsir-template" class="form-label">{{ $t("Template ID") }}</label>
```

Comment:
could you add all new translation keys to `en.json`?
Otherwise, you will not be able to translate this in weblate 😉